src/main/java/com/thevoxelbox/voxelsniper/brush/type/performer/splatter/SplatterOverlayBrush.java

```diff
@@ -107,6 +111,15 @@ public void handleCommand(String[] parameters, Snipe snipe) {
         }
     }
 
+    @Override
+    public List<String> handleCompletions(String[] parameters, Snipe snipe) {
+        if (parameters.length == 1) {
+            String parameter = parameters[0];
+            return super.sortCompletions(Stream.of("d", "all", "some", "s", "g", "r", "randh", "yoff"), parameter, 0);
```

Comment:
Starting the args with `s` or `r` would complete some or randh. Individual letters aren't treated properly.

Reply:
I don't know what to do here 😓 the handler is only suggesting an alternative. Should nothing be suggested if an existing valid argument is given?

Reply:
Are `s` and `r` aliases some & randh?

Reply:
No

Reply:
Oic,, the existing behavior must be sufficient then without changing existing 1 letter args. Tho that could be better for sake intelligibility.